.github/workflows/release.yaml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+on:
+  push:
+    tags: "*"
+    branches: master
+
+jobs:
+  version-match:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install yq
+        run: sudo snap install yq
+      - name: Ensure version equality
+        run: if [[ "$(echo $GITHUB_REF | sed s+refs/tags/++)" == "" ]] || [[ "$(echo $GITHUB_REF | sed s+refs/tags/++)" != "$(yq r helm/values.yaml 'deployment.image' | cut -d ':' -f2)" ]]; then exit 1; fi
+
+  integration-test:
+    runs-on: ubuntu-latest
+    needs: [version-match]
+    steps:
+      - uses: actions/checkout@v2
+
+  docker_build_sign_push:
+    runs-on: ubuntu-latest
+    container:
+      image: docker:stable
+    needs: [integration-test]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Prepare docker content trust
+        run: mkdir -p ~/.docker/trust/private && echo "${{ secrets.DOCKER_KEY }}" > ~/.docker/trust/private/${{ secrets.DOCKER_KEY_NAME }}
+      - name: Build image
+        run: docker build -t $(yq r helm/values.yaml 'deployment.image') -f docker/Dockerfile .
```

Comment:
would opt for building with DCT and force us to use a signed base image. What do you think?

Reply:
Would say that's definetly out of scope for this PR.
Afaik DCT isnt spread far enough that we could just do that, except if we signed our own copies of some base image. That would make the build less trustworthy from an outsider's point though...